concent_api/gatekeeper/enum.py

```diff
@@ -0,0 +1,6 @@
+import enum
+
+
+@enum.unique
+class HashingAlgorytm(enum.Enum):
```

Comment:
`HashingAlgorithm`

Reply:
Also, please rename `enum.py` to `enums.py`. That's the usual convention in Django.